CourseGrab/CoursesViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CoursesViewController.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 1/26/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CoursesViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        navigationItem.title = "CourseGrab"
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "settingsIcon"),
```

Comment:
For both of these bar button items, fix the spacing/indentation. I think it should be:
```
navigationItem.leftBarButtonItem = UIBarButtonItem(
    image: UIImage(named: "settingsIcon"),
    style: .plain,
    target: self,
    action: nil
)
```
Similar thing for the right bar button item.